core/client/app/templates/setup/two.hbs

```diff
@@ -18,22 +18,24 @@
             </div>
             <a class="edit-account-image" href="#"><i class="icon-photos"><span class="sr-only">Upload an image</span></i></a>
         </figure>
-        <div class="form-group">
+        {{#gh-form-group errors=errors prop="email"}}
             <label for="email-address">Email address</label>
             <span class="input-icon icon-mail">
-                {{input type="email" name="email" placeholder="Eg. john@example.com" class="gh-input" autofocus="autofocus" autocorrect="off" value=email}}
+                {{gh-input type="email" name="email" placeholder="Eg. john@example.com" class="gh-input" autofocus="autofocus" autocorrect="off" value=email model=this prop="email"}}
```

Comment:
Yeah I get that. That was the quick and dirty way to do it, but I think I just figured out a way to do it better.